README.md

```diff
@@ -26,6 +26,7 @@ under the License.
 [![License](http://img.shields.io/:license-Apache%202-blue.svg)](http://www.apache.org/licenses/LICENSE-2.0.txt)
 [![PyPI - Python Version](https://img.shields.io/pypi/pyversions/apache-airflow.svg)](https://pypi.org/project/apache-airflow/)
 [![Twitter Follow](https://img.shields.io/twitter/follow/ApacheAirflow.svg?style=social&label=Follow)](https://twitter.com/ApacheAirflow)
+[![Slack Status](https://img.shields.io/badge/slack-join_chat-white.svg?logo=slack&style=social)](https://apache-airflow-slack.herokuapp.com/)
```

Comment:
Any particular reason to have the white one? The background is white as well, so maybe the default green one might have some more contrast.

Reply:
It doesn't make any difference because it is a `social` badge.

White: https://img.shields.io/badge/slack-join_chat-white.svg?logo=slack&style=social
Green: https://img.shields.io/badge/slack-join_chat-green.svg?logo=slack&style=social

Reply:
Good point, I think I've lost part of the url when copying.